src/views/Login/Login.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Logo from '../../assets/images/Respond_Logo_icon_fullcolor.png'
+import Footer from "../../components/Footer/Footer"
+import './Login.css';
+
+export default class CreateAccount extends React.Component {
```

Comment:
@rrsampathi  is it type between CreateAccount or LoginPage?

Reply:
Ah sorry, forgot to switch the class name to Login. Good catch!